triage/templates/triage/custom-page.html

```diff
@@ -40,6 +41,102 @@ <h2 class="heading-large">Connect with international buyers</h2>
   </div>
 </section>
 {% endif %}
+{% if persona.name == 'OCCASIONAL_EXPORTER'%}
+<section>
+  <div class="container">
+    <div class="grid-row">
+      <div class="column-half">
+        <h2 class="heading-large">Explore new markets</h2>
+        <p>Learn about the best markets for your sector, increase your exporting knowledge and start connecting with overseas buyers.</p>
+        {% if not sso_is_logged_in %}
+          <p><a class="link" href="{{ sso_register_url }}">Register</a> or <a class="link" href="{{ sso_login_url }}">sign in</a> to save your progress.</p>
+        {% endif %}
+      </div>
+    </div>
+  </div>
+</section>
+{% endif %}
+{% if triage_result.is_exporting_goods %}
+<section id="sector">
+  <div class="container">
+    <div class="grid-row">
+      <div class="column-half">
+        <form action="/custom/" method="post">
```

Comment:
if this template is only used on `/custom/` then this can be blank.

if the template is used in other places too it's better to use` {% url '<the-url-name' %}`, which you can get from urls..py